lib/runner.js

```diff
@@ -146,7 +146,9 @@ function runner(script, payload, options) {
   //
   let runnerPlugins = [];
   _.each(runnableScript.config.plugins, function tryToLoadPlugin(pluginConfig, pluginName) {
-    let requireString = 'artillery-plugin-' + pluginName;
+    let pluginPackage = pluginConfig.package || runnableScript.config.pluginsPackage;
+    let pluginPrefix = pluginPackage ? pluginPackage + '/' : 'artillery-plugin-';
```

Comment:
Should we just rename as below to avoid the misleading factor?

pluginConfig.package || runnableScript.config.pluginsPackage 
to 
pluginConfig.pluginPrefix || runnableScript.config.pluginsPrefix